spring-cloud-dataflow-rest-client/src/main/java/org/springframework/cloud/dataflow/rest/client/DataFlowTemplate.java

```diff
@@ -90,6 +90,7 @@ public DataFlowTemplate(URI baseURI, RestTemplate restTemplate) {
 		Link link = getLink(resourceSupport, "tasks");
 		resources.put("tasks/definitions", new UriTemplate(link.getHref() + "/definitions"));
 		resources.put("tasks/deployments", new UriTemplate(link.getHref() + "/deployments"));
+		resources.put("tasks/executions", new UriTemplate(link.getHref() + "/executions"));
```

Comment:
Opened https://github.com/spring-cloud/spring-cloud-dataflow/issues/306 to refactor the way resource support is handled by tasks.